loop-server/manifest.json

```diff
@@ -2,16 +2,30 @@
   "envs" : {
     "stage": {
       "test_types": [ "stack-check", "loadtest" ],
+      "stack_label": "loopsvrstg",
+      "aws_region": "eu-west-1",
       "urls": {
 	"root" : "loop.stage.mozaws.net",
-	"push_config" : "loop.stage.mozaws.net/push-server-config"
+	"push_config" : "loop.stage.mozaws.net/push-server-config",
+	"heartbeat" : "loop.stage.mozaws.net/__heartbeat__"
+      }
+    },
+    "pre-prod": {
+      "test_types": [ "stack-check" ],
+      "aws_region": "eu-west-1",
+      "urls": {
+        "root": "loop.services.mozilla.com",
+	"push_config" : "loop.services.mozilla.com/push-server-config",
+	"heartbeat" : "loop.services.mozilla.com/__heartbeat__"
```

Comment:
in most cases they'd probably be the same, but a pre-prod test has some /etc/hosts setup step that the prod test would not.  also, i'm not sure about [security] test type(s).  it may be that there will be a few security test types we can throw against a pre-prod env, but probably wouldn't want to do against our live prod env.